### PR TITLE
feat: NomadNet browser with Micron rendering, URL bar, and URI handler

### PR DESCRIPTION
**Pull request by torlando-tech**
## Summary

- **NomadNet page browser** — browse NomadNet nodes via Reticulum with full Micron markup rendering (text styling, colors, links, forms, ASCII art, partial auto-refresh)
- **Micron parser module** — standalone parser for NomadNet's Micron markup language with comprehensive test coverage (878-line test suite)
- **Editable URL bar** — tap the address bar to navigate to any `hash:/path`, with Copy URL and Share menu items
- **`nomadnetwork://` URI handler** — external apps can open NomadNet pages or LXMF conversations via deep links
- **Custom scheme linkification** — `nomadnetwork://` and `lxma://` URLs are now clickable in conversation messages
- **Page caching** — file-based cache with configurable TTL for previously visited pages
- **Pinch-to-zoom** — layout-aware scaling in monospace scroll mode
- **Identify to node** — reveal identity to node operators for authenticated pages
- **Partial support** — async loading and auto-refresh of page partials

## Test plan

- [x] Unit tests: 617 lines covering ViewModel (URL helpers, navigation, lxmf@ routing, history, forms, identify)
- [x] Unit tests: 878 lines covering Micron parser (styling, colors, links, fields, alignment, edge cases)
- [x] Unit tests: NomadNetPageCache and URL resolution
- [ ] Manual: open NomadNet browser, verify URL bar shows `hash:/path`
- [ ] Manual: tap URL bar, edit, press Go — navigates to new page
- [ ] Manual: overflow menu Copy URL / Share works
- [ ] Manual: `adb shell am start -a android.intent.action.VIEW -d "nomadnetwork://hash:/page/index.mu"` opens browser
- [ ] Manual: `nomadnetwork://` links in conversations are clickable and open browser
- [ ] Manual: `lxmf@hash` links on Micron pages open conversation screen

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: micron/src/main/java/com/lxmf/messenger/micron/MicronColor.kt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        private const val MAX_GRAYSCALE = 99
+
+        /**
+         * Parse a Micron color string (3 chars after F/B command).
+         * - "ddd" → Hex(0xDD, 0xDD, 0xDD)
+         * - "g50" → Grayscale(50)
+         */
+        @Suppress("ReturnCount")
+        fun parse(colorStr: String): MicronColor? {
+            if (colorStr.length < 3) return null
```
Comment (Some-O-ne) [inline micron/src/main/java/com/lxmf/messenger/micron/MicronColor.kt:51]: where truecolor `FT and `BT :<

**Comment by Some-O-ne** [comment]
heeey torlando when truecolor color tags? :>

File: micron/src/main/java/com/lxmf/messenger/micron/MicronColor.kt (hunk 1 above)
Comment (torlando-tech) [reply micron/src/main/java/com/lxmf/messenger/micron/MicronColor.kt:51]: Thank you for the reminder!

**Comment by torlando-tech** [comment]
> heeey torlando when truecolor color tags? :>

Will have to get to it tomorrow. I'm gonna have to do another round of testing after all the automated iteration this went through anyway
